uqcsbot/scripts/events.py

```diff
@@ -161,6 +177,43 @@ def get_current_time():
     return datetime.now(tz=BRISBANE_TZ).astimezone(utc)
 
 
+def handle_calendar(calendar) -> List[Event]:
+    """
+    returns a list of events from a calendar
+    """
+    events = []
+    current_time = get_current_time()
+    # subcomponents are how icalendar returns the list of things in the calendar
+    for c in calendar.subcomponents:
```

Comment:
Single letter variable names, if used at all, should be used over tiny scopes only. Something like `component` might make this less logically dense to unpack for people who have to read it later. Though I see they've been used liberally already in this file, so up to you as to whether it's worth addressing.

Reply:
Thought i might as well fix this while i was there